src/importexport/mei/internal/meiconverter.cpp

```diff
@@ -2806,6 +2807,10 @@ libmei::StaffDef Convert::staffToMEI(const engraving::Staff* staff)
     if (staffType) {
         meiStaffDef.SetLines(staffType->lines());
     }
+    // @lines.color
+    if (staff->staffType(engraving::Fraction(0, 1))->color() != engravingConfiguration()->defaultColor()) {
+        meiStaffDef.SetLinesColor(staff->staffType(engraving::Fraction(0, 1))->color().toString());
```

Comment:
Can't you reuse the `staffType` variable?
```suggestion
    if (staffType->color() != engravingConfiguration()->defaultColor()) {
        meiStaffDef.SetLinesColor(staffType->color().toString());
```